pom.xml

```diff
@@ -450,6 +451,24 @@ under the License.
 				<version>${log4j.version}</version>
 			</dependency>
 
+			<dependency>
+				<groupId>org.slf4j</groupId>
+				<artifactId>log4j-over-slf4j</artifactId>
+				<version>${slf4j.version}</version>
+			</dependency>
+
+			<dependency>
+				<groupId>ch.qos.logback</groupId>
+				<artifactId>logback-classic</artifactId>
+				<version>${logback.version}</version>
+			</dependency>
+
+			<dependency>
+				<groupId>ch.qos.logback</groupId>
+				<artifactId>logback-core</artifactId>
+				<version>${logback.version}</version>
+			</dependency>
```

Comment:
This section indicates what versions all other dependencies should be using. Since other libraries depend on but exclude log4j, we still need to define those here. We now also need to include logback